src/main/kotlin/org/rust/cargo/project/toolwindow/CargoProjectsTree.kt

```diff
@@ -48,7 +49,7 @@ open class CargoProjectsTree : SimpleTree() {
                 val cargoProject = selectedProject ?: return
                 // Capitalize command name to be consistent with line market providers
                 // TODO: not to use `capitalize` here
```

Comment:
Is this Todo fixed now?

Reply:
:thinking: @Undin ?

Reply:
Actually, no. Let's update name of function in comment